File: web3rpc/rpc-specs/code-samples/javascript/debug/logging/verbosityByID.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");    
    const id = 1;
    const level = 3;

    provider.debug.verbosityByID(id, level, {}, (err, data, response) => {
        console.log(data);
    });
}
)()

File: web3rpc/rpc-specs/code-samples/python/debug/logging/verbosityByID.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

verId = 1
level = 3

w3 = Web3(Web3.HTTPProvider(host))
debug_response = w3.debug.verbosity_by_id(verId, level)

print(debug_response)
